js_modules/dagit/packages/core/src/pipelines/GraphExplorer.tsx

```diff
@@ -291,8 +288,8 @@ export const GraphExplorer: React.FC<GraphExplorerProps> = (props) => {
           <Route
             // eslint-disable-next-line react/no-children-prop
             children={({location}: {location: any}) => (
-              <SidebarTabbedContainer
```

Comment:
I renamed this because 1) it's the thing rendered by a Route so it's sort of a root, and I think we should avoid calling in the sidebar "Container" because "SolidContainer" is actually an interface exposed by the GraphQL api